PR: https://github.com/odoo/odoo/pull/

From: 7978d4ba0f093c4b5400dbb1776ff5c0416e203b
From: Yannick Tivisse
Date: 2018-09-10 12:23:42

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'res.users'
-    has_group_manage_vendor_price = fields.Boolean(
-    has_group_warning_purchase = fields.Boolean(
-    group_purchase_user = fields.Selection(

Total Changes: 23

Revert "[IMP] *: Define groups on res.users models"

This reverts commit 99f497b390e680629dce3a7e2e282ada74ee9884.

================================= pseudo patch: =================================

--- a/addons/purchase/models/__init__.py
+++ b/addons/purchase/models/__init__.py
@@ -7,4 +7,3 @@ from . import product
 from . import res_company
 from . import res_config_settings
 from . import res_partner
-from . import res_users

--- a/addons/purchase/models/res_users.py
+++ b/None
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    has_group_manage_vendor_price = fields.Boolean(
-        'Manage Vendor Price', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='purchase.group_manage_vendor_price')
-
-    has_group_warning_purchase = fields.Boolean(
-        'A warning can be set on a product or a customer (Purchase)',
-        compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='purchase.group_warning_purchase')
-
-    group_purchase_user = fields.Selection(
-        selection=lambda self: self._get_group_selection('base.module_category_purchase_management'),
-        string='Purchases', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        category_xml_id='base.module_category_purchase_management')
